src/styles/main.scss

```diff
@@ -0,0 +1,132 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style-type: none;
+}
+
+$width-clock: 600px;
+$rotate: "1"30deg,
+  "2"60deg,
+  "3"90deg,
+  "4"120deg,
+  "5"150deg,
+  "6"180deg,
+  "7"210deg,
+  "8"240deg,
+  "9"270deg,
+  "10"300deg,
+  "11"330deg;
+
+.wrapper {
+  width: 100vw;
+  height: 100vh;
+  background-color: rgb(50, 177, 227);
+  display: flex;
+  justify-content: center;
+  align-items: center;

```

Comment:
Please pay attention to your CSS properties order
https://9elements.com/css-rule-order/
Fix all cases

Reply:
fix it, please
https://github.com/mate-academy/layout_stop-watch/pull/1949/files#r1067896760